site/content/en/docs/api_sdk/sdk/developer-guide.md

```diff
@@ -19,55 +19,29 @@ the repository. To get the full package, one need to generate missing package fi
 
 ## How to generate package code
 
-1. Obtain the server API schema
+1. Install generator dependencies:
```

Comment:
For what purpose?

Reply:
In this case, the CI check prints the command you need to run. There's also a VS code launch configuration that does it.

Reply:
OK

Reply:
There is one problem: The schema may not be generated correctly If we have an outdated image.

Reply:
Agree, I've also faced this difference in the generated schema.

Reply:
I have found the cause for this discrepancy; it's because in the Docker image, the production configuration is selected, which uses a different database. I'll submit a fix to the VS code configuration.